TheMovieDb/Views/GeneriMovieCollectionView/GenericCollectionViewCells/LargeCollectionViewCell.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  LargeCollectionViewCell.swift
+//  TheMovieDb
+//
+//  Created by Jonathan Hernandez on 10/11/21.
+//
+
+import UIKit
+import Kingfisher
+
+class LargeCollectionViewCell: UICollectionViewCell {
+    static let reuseIdentifer = "large-movie-cell-reuse-identifier"
+    let titleLabel = UILabel()
+    let portraitImageView = UIImageView()
+    let contentContainer = UIView()
+    
+    var title: String? {
+        didSet {
+            configure()
```

Comment:
I have my doubts about this configure() method each time you assign these properties.